modules/gitops-openshift-go-terminology.adoc

```diff
@@ -0,0 +1,47 @@
+Declarative setup:: A declarative description of the infrastructure you need in your specified environment, for system and application setup or configuration. You can specify this description in a YAML configuration file in the Git repository. The declarative setup contains an automated process to make your environment and infrastucture match the described state. Examples include defining Argo CD applications, projects and settings declaratively using YAML manifests.
+Live state:: The live state of an application. Examples include what pods and other application resources are deployed.
+Target state:: The desired state of an application, as represented by files in a Git repository.
+Sync:: The process of making an application move to its target state. Examples include syncing an application by applying changes to a cluster.
+Sync status:: The status of an application that indicates whether the live state matches the target state.
+Refresh:: The process to compare the latest code in Git with the live state and figure out the difference.
+AppProject:: A resource object and CRD representing a logical grouping of applications within a project, which governs where and how an application might manage resources. 
+Argo CD instance:: A copy or an instantiation that encapsulates all of the stateful aspects of a running Argo CD.
+Instance scopes:: Modes that determine how you can want to operate an Argo CD instance. The available modes are cluster-scoped instance and namespace-scoped instance.
+Namespace-scoped instance (Application delivery instance):: A mode wherein Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
+Cluster-scoped instance:: A mode wherein Argo CD is configured to manage every single resource on the cluster including certain cluster-specific resources such as platform Operators, optional OLM Operators, and secrets.
+Default cluster-scoped instance:: A default instance that a {gitops-title} Operator instantiates immediately after its installation, in the `openshift-gitops` namespace.
+ApplicationSet (Argo CD application set):: A resource object and a CRD that automatically generates Argo CD applications based on the contents of an `ApplicationSet` CR. Cluster administrators use this CRD to define a single `ApplicationSet` CR to generate and update multiple corresponding Argo CD `Application` CRs.
+Argo CD API server:: A gRPC/REST server that exposes the API consumed by the Web UI, CLI, continuous integration (CI), and continuous deployment (CD) systems. 
+Ardo CD server (Argo CD-server):: A server that serves the API and UI for Argo CD.
+Argo CD repository server (Argo CD-repo-server):: A server that reads from Git repositories that hold the application manifests, generates manifests, and executes custom configuration management tooling. 
```

Comment:
> Argo CD repository server (Argo CD-repo-server)::

How about this:

Argo CD repository server (Argo CD-repo-server):: An Argo CD component that reads from Git repositories, generates corresponding application manifests, executes custom configuration management tooling, and returns the result to application controller.